app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -95,6 +90,9 @@ export default {
     ...mapActions([`updateDelegates`, `queryWalletBalances`]),
     showModal(denomination) {
       this.$refs.sendModal.open(denomination)
+    },
+    async faucet() {
+      await this.$store.dispatch(`getMoney`, this.wallet.address)
```

Comment:
```suggestion
      await this.$store.dispatch(`getMoney`, this.session.address)
```

we get rid of `wallet.address`